OJP_StopEvents.xsd

```diff
@@ -123,7 +124,7 @@
 			</xs:element>
 			<xs:element name="IncludeStopHierarchy" type="HierarchyEnumeration" default="no" minOccurs="0">
```

Comment:
"local" was defined as default in the documentation but the actual in the attribute still is `default="no"`